esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -400,8 +398,8 @@ impl<'d> Rtc<'d> {
         h.write(|w| unsafe { w.bits((boot_time_us >> 32) as u32) });
     }
 
-    /// Get the current time.
-    pub fn current_time(&self) -> NaiveDateTime {
+    /// Get the current time in microseconds.
```

Comment:
Could you expand this documentation to show how to use it with `jiff`?

Reply:
How? Construct a `jiff::Timestamp` with it? Get timezones involved and calculate the day of the week?